src/test/java/seedu/address/storage/JsonSerializableTutorsPetTest.java

```diff
@@ -55,10 +93,46 @@ public void toModelType_invalidClassFile_throwsIllegalValueException() throws Ex
         assertThrows(IllegalValueException.class, dataFromFile::toModelType);
     }
 
+    /**
+     * Ensures that Tutor's Pet will not be able to boot up although Java tries
+     * to pad missing {@code UUID} digits in broken {@code UUID}s with zeros.
+     */
+    @Test
+    public void toModelType_invalidStudentUuidInClassFile1_throwsIllegalValueException() throws Exception {
+        JsonSerializableTutorsPet dataFromFile = JsonUtil.readJsonFile(INVALID_STUDENT_UUID_IN_CLASS_FILE_1,
+                JsonSerializableTutorsPet.class).get();
+        assertThrows(IllegalValueException.class, dataFromFile::toModelType);
+    }
+
+    /**
+     * Ensures that Tutor's Pet will not be able to boot up given an incomplete {@code UUID}.
+     * (e.g "0c527a3f-8a6f-4c16-b57d-")
+     */
```

Comment:
I am not sure about this... But should the full-stop be after the example?
```suggestion
    /**
     * Ensures that Tutor's Pet will not be able to boot up given an incomplete {@code UUID}
     * (e.g "0c527a3f-8a6f-4c16-b57d-").
     */
```